WTWSMS/history/provinces/666 - Gilan.txt

```diff
@@ -16,8 +16,8 @@ b_masouleh = castle
 #b_rudbar = castle
 
 #Misc
-culture = aramean
-religion = manichean
+culture = kurdish
+religion = yazidi
```

Comment:
Yazidis long preceded Islam. There was no connection until the Muslims crushed them.